examples/matplotlib/mpl_plot_pair_point_estimate.py

```diff
@@ -0,0 +1,24 @@
+"""
+Point Estimate Pairplor
```

Comment:
Hi @agustinaarroyuelo I was checking out your new examples and noticed your docstrings for the example `.py` files say Pairplo**r** instead of what I assume should be Pairplo**t**.

And the files don't end in newlines, which PEP8 doesn't like

Reply:
Hi, thanks for letting me know!